package.json

```diff
@@ -106,7 +106,6 @@
         "clearMocks": true,
         "restoreMocks": true,
         "moduleNameMapper": {
-            "react-intl": "<rootDir>/scripts/jest/mocks/react-intl-mocks.js",
```

Comment:
line 47 exclusion for __mocks__

Reply:
unless you want to move it under __tests__

Reply:
I'm removing that commit, that'll be on #1391